modules/acquia_cms_tour/src/Form/AcquiaConnectorForm.php

```diff
@@ -77,11 +77,21 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       ];
       if (isset($module_info['configure'])) {
         $form[$module]['actions']['advanced'] = [
+          '#prefix' => '<div class= "dashboard-tooltiptext">',
           '#markup' => $this->linkGenerator->generate(
             'Advanced',
             Url::fromRoute($module_info['configure'])
           ),
-          '#suffix' => "</div>",
+          '#suffix' => '</div>',
+        ];
+        $form[$module]['actions']['advanced']['information'] = [
```

Comment:
@panshulK Can you please check if we can move all of this common code in `InstallationWizardForm.php` ? so that we don't need to repeat in all configuration forms.

Reply:
@vishalkhode1 The installationWizard is a separate form that opens form in a pop up. The ticket requires displaying Advanced button on individual forms on /admin/tour/dashboard page itself. Hence we cannot move the code to InstallationWizardForm.php.